src/HDE/hde32.c

```diff
@@ -15,7 +14,11 @@ unsigned int hde32_disasm(const void *code, hde32s *hs)
     uint8_t *ht = hde32_table, m_mod, m_reg, m_rm, disp_size = 0;
 
     // Avoid using memset to reduce the footprint.
+#ifdef MINGW
+    memset((LPBYTE)hs, 0, sizeof(hde32s));
+#else
     __stosb((LPBYTE)hs, 0, sizeof(hde32s));
```

Comment:
As __stosb is MS specific intrinsic I think is save to replace __stosb with memset in both cases.